Plugin/Model/ConfigPlugin.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
```

Comment:
@LordZardeck - All file docblocks need to be formatted consistently with the ones already existing in the extension.